ci/tasks/deploy.sh

```diff
@@ -82,8 +78,7 @@ resource_pools:
 - name: vms
   network: private
   stemcell:
-    url: $STEMCELL_URL
-    sha1: $STEMCELL_SHA
+    url: file://stemcell.tgz
```

Comment:
@sedouard Remove the URL avoid to keep two sources in CI pipeline. After the official stemcell is ready, it is more easier to switch.